recipe/meta.yaml

```diff
@@ -42,11 +44,17 @@ requirements:
     - {{ cdt('fontconfig-devel') }}      # [linux and tk_variant == 'xft']
     - {{ cdt('libXrender-devel') }}      # [linux and tk_variant == 'xft']
     - {{ cdt('libXft-devel') }}          # [linux and tk_variant == 'xft']
+    - {{ cdt('libpng-devel') }}          # [linux and tk_variant == 'xft']
+    - {{ cdt('libuuid-devel') }}         # [linux and tk_variant == 'xft']
+    - {{ cdt('expat-devel') }}           # [linux and tk_variant == 'xft']
     - pkg-config                         # [linux and tk_variant == 'xft']
   host:
-    - zlib  # [unix]
+    - zlib
 
 test:
+  requires:
+    # needs to run emulated tests
+    - bzip2  # [build_platform != target_platform]
```

Comment:
This is very curious. 

If this is not needed by the native builds, it must be finding libbzip2 from some other environment and adopting it right?